Modules/Loadable/VolumeRendering/MRML/vtkMRMLShaderPropertyNode.cxx

```diff
@@ -0,0 +1,157 @@
+// MRML includes
+#include "vtkMRMLScene.h"
+#include "vtkMRMLShaderPropertyNode.h"
+#include "vtkMRMLVolumePropertyStorageNode.h"
+
+// VTK includes
+#include <vtkCommand.h>
+#include <vtkIntArray.h>
+#include <vtkObjectFactory.h>
+#include <vtkShaderProperty.h>
+#include <vtkUniforms.h>
+
+// STD includes
+#include <algorithm>
```

Comment:
Do you really need algorithm, cassert, and limits?

Reply:
No, I removed this